### ISSUE TITLE
Can't connect to postgres

### ISSUE THREAD

**Issue by vi1world** [grafbase/grafbase#1777]
Hello,
I tried to connect to postgres via the cli but it just gives me this error. The connection string works when I connect it using a database explorer. 

**Describe the bug**
`Error: Postgres data source: invalid type: null, expected a string`

**To Reproduce**
`import { config, connector, graph } from '@grafbase/sdk'

const g = graph.Standalone()

const pg = connector.Postgres('pg', {
  url: g.env('DATABASE_URL'),
})

g.datasource(pg)

export default config({
  graph: g,
  cache: {
    rules: [
      {
        types: ['Query'],
        maxAge: 60,
        staleWhileRevalidate: 60,
      },
    ],
  },
})
`

**Comment by fbjork** [comment]
Hi @vi1world! What does format does the connection string have? `postgres://user:password@hostname:port/schema` is what we expect

**Comment by vi1world** [comment]
Yes my connection string starts with postgres://
I also tried postgresql:// as well
It could connect to the database because when I tried with the wrong credentials it doesn't work. 
When I try with the correct credentials, it gets this error `Error: pg data source: invalid type: null, expected a string`

**Comment by fbjork** [comment]
> Yes my connection string starts with postgres:// I also tried postgresql:// as well It could connect to the database because when I tried with the wrong credentials it doesn't work. When I try with the correct credentials, it gets this error `Error: pg data source: invalid type: null, expected a string`

How are you setting the DATABASE_URL environment variable?

**Comment by tomhoule** [comment]
I am investigating this. It would be very helpful to know the database schema and / or the postgres version you are using, if you can't share the database schema.

**Comment by vi1world** [comment]
I tried again today with a different postgres connection. This time with supabase: 
The connection string looks like this
`postgresql://<username>:<password>@aws-0-ap-southeast-1.pooler.supabase.com:6543/postgres`

I got this error when running:

 npx grafbase dev
Grafbase CLI 0.75.0

`Error: pg data source: could not connect to Postgres: error performing TLS handshake: invalid peer certificate: UnknownIssuer`

The connection string works fine when I tried with Hasura. But just doesn't work with Grafbase for some reason. Also tried `?sslmode=disable` as well.
